.env

```diff
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_ADMIN_URL=https://samenvvvv.com
```

Comment:
Please remove `.env` file and use `.env.local` instead, which is already included in gitignore